### PR TITLE
Fix win port

### PR DESCRIPTION
**Pull request by petrovigor**
Built on windows. Prepared to developening.

### CODE REVIEW

File: core/actor-zeta/messaging/message_header.hpp [hunk 1]
```diff
@@ -17,9 +17,9 @@ namespace actor_zeta {
 
             message_header &operator=(const message_header &) = default;
 
-            message_header(message_header &&) noexcept = default;
+            message_header(message_header &&) = default;
```
Comment (kotbegemot) [inline core/actor-zeta/messaging/message_header.hpp]: What is the reason to remove noexcept ?

File: core/actor-zeta/messaging/message_header.hpp [hunk 2]
```diff
@@ -17,9 +17,9 @@ namespace actor_zeta {
...
 
-            message_header &operator=(message_header &&) noexcept = default;
+            message_header &operator=(message_header &&) = default;
```
Comment (kotbegemot) [inline core/actor-zeta/messaging/message_header.hpp]: What is the reason to remove noexcept ?

Comment (petrovigor) [reply core/actor-zeta/messaging/message_header.hpp]: The reason is MVC compile errors (function signature incompatibility with and without keyword 'noexcept' in implementation). This two lines is my flub. Already fixed in new commit.

**Comment by kotbegemot** [comment]
@petrovigor  please read : 
* https://stackoverflow.com/questions/37276413/default-move-constructor-assignment-and-deleted-copy-constructor-assignment
* https://en.cppreference.com/w/cpp/language/noexcept_spec

**Pull request closed without merging**